### PR TITLE
[Feature] Add props config for heading

### PR DESCRIPTION
**Pull request by Qadra42**
I added propsConfig in order to improve customization.
Screenshot 2023-06-15 at 13 17 51

### CODE REVIEW

File: src/components/calendarPanel.tsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ export const CalendarPanel: React.FC<CalendarPanelProps> = ({
                 getBackProps={getBackProps}
                 propsConfigs={propsConfigs}
               />
-              <Heading size="sm" minWidth={'5rem'} textAlign="center">
+              <Heading size="sm" minWidth={'5rem'} textAlign="center" {...propsConfigs?.headingProps}>
```
Comment (aboveyunhai) [inline src/components/calendarPanel.tsx:102]: maybe `dateHeadingProps` and `weekdayLabelProps`?
a little bit of verbose but clearer?

Comment (aboveyunhai) [reply src/components/calendarPanel.tsx:102]: also, thx for the PR.

Comment (Qadra42) [reply src/components/calendarPanel.tsx]: > maybe `dateHeadingProps` and `weekdayLabelProps`? a little bit of verbose but clearer?

For sure!

**Comment by aboveyunhai** [comment]
@Qadra42  0.2.9 is out, let me know if there is an issue.

**Pull request merged**